packages/docs/docs/graphql/basic-queries.mdx

```diff
@@ -50,6 +50,10 @@ For example, to request a `Patient` by ID:
 
 This query retrieves the `resourceType`, `id`, `name`, and `address` of the specified `Patient`.
 
+:::note Access Policies
+When using GraphQL, [access policies](/docs/access/access-policies) are enforced, so users will not be able to read or edit any resources they do not have access to.
```

Comment:
If we wanted to be extra precise here, I'd suggest the following — feel free to take or leave it at your discretion

```suggestion
When using GraphQL, [access policies](/docs/access/access-policies) are enforced, so users will not be able to read or edit any resources (or inner fields) they do not have access to.
```